guide/14-deep-learning/exploring_model_extensibility_for_named_entity_recognition.ipynb

```diff
@@ -0,0 +1,722 @@
+{
```

Comment:
<p>Use&nbsp;<strong>ESRI Model Definition (.emd) file&nbsp;</strong>everywhere&nbsp;<strong>emd</strong>&nbsp;is written.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2112/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Done</p>